lib/v2/uestc/jwc.js

```diff
@@ -0,0 +1,58 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+const dateRegex = /(20\d{2})\/(\d{2})\/(\d{2})/;
+
+const baseUrl = 'https://www.jwc.uestc.edu.cn/';
+const detailUrl = 'https://www.jwc.uestc.edu.cn/info/';
+
+const map = {
+    important: 'hard/?page=1',
+    student: 'list/256/?page=1',
+    teacher: 'list/255/?page=1',
+    teach: 'list/40/?page=1',
+    office: 'list/ff80808160bcf79c0160c010a8d20020/?page=1',
+};
+
+module.exports = async (ctx) => {
+    const type = ctx.params.type || 'important';
+    const pageUrl = map[type];
+    if (!pageUrl) {
+        throw new Error('type not supported');
+    }
+
+    const response = await got({
+        method: 'get',
+        url: baseUrl + pageUrl,
+        headers: {
+            Referer: baseUrl,
+        },
+    });
```

Comment:
Use either one convention, but not both
https://github.com/DIYgod/RSSHub/blob/8e334b9b0ec4356befb3bb620d5102c301a7eea9/lib/v2/uestc/gr.js#L9